source/help/settings/channel-settings.rst

```diff
@@ -38,6 +38,14 @@ the channel topic or provide links to frequently accessed documents. Any
 channel member can edit this setting, unless the System Administrator
 has `restricted the permissions <https://docs.mattermost.com/administration/config-settings.html#enable-public-channel-renaming-for>`__.
 
+Adding links to the Channel Header
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add links to the channel header using markdown.
+
```

Comment:
I think it's a stronger sentence if you write:

*Use Markdown to add links to the channel header.*

or even

*You can use Markdown to add links to the channel header.*

But then that raises the question: Is all Markdown formatting available in the header, or just linking?